app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    dict_info_by_name = {}
+    for current_person in people:
+        Person(
+            current_person["name"],
+            current_person["age"]
+        )
+        dict_info_by_name[current_person["name"]] = current_person
+
+    for person_cls_obj in Person.people.keys():
```

Comment:
```suggestion
    for person_name in Person.people:
```
a bit confusing naming